### PR TITLE
Create script to download WB datasets.

### PR DESCRIPTION
**Pull request by keyurva**

### CODE REVIEW

File: scripts/world_bank/datasets/datasets.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
+import requests
```
Comment (ajaits) [inline scripts/world_bank/datasets/datasets.py:1]: Can you add the copyrigth header
Pls also add a file level block comment/readme on how to run the script.

File: scripts/world_bank/datasets/datasets.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+                download_urls.append(download_url)
+
+        with multiprocessing.Pool(POOL_SIZE) as pool:
+            pool.starmap(download, zip(download_urls))
+
+        logging.info('# files downloaded: %s', len(download_urls))
+
+
+def download(url):
+    file_name = re.sub(r'[^a-zA-Z0-9.\-_]', '', url.split('/')[-1])
```
Comment (ajaits) [inline scripts/world_bank/datasets/datasets.py:66]: move '-' to the end of the char list: `[^a-zA-Z0-9.\_-]`

Comment (ajaits) [inline scripts/world_bank/datasets/datasets.py:66]: escape `.`: `[^a-zA-Z0-9\.\-_]`

File: scripts/world_bank/datasets/datasets.py (hunk 1 above)
Comment (keyurva) [reply scripts/world_bank/datasets/datasets.py:27]: Done.

File: scripts/world_bank/datasets/datasets.py (hunk 2 above)
Comment (keyurva) [reply scripts/world_bank/datasets/datasets.py]: Done.

File: scripts/world_bank/datasets/datasets.py [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+
+
+def load_json_file(json_file):
+    with open(json_file, 'r') as f:
+        return json.load(f)
+
+
+def main(_):
+    fetch_and_write_datasets_csv()
+    # download_datasets()
```
Comment (ajaits) [inline scripts/world_bank/datasets/datasets.py:236]: can we add a command line flag for this so user won't have to edit the script to download datasets?

Example:
https://docs.google.com/document/d/1asu3CdLHd4xZZIb7oLmVdiaMOO3EQl7BYvD6c0d41pU/edit#heading=h.6mh8op9lb42r

File: scripts/world_bank/datasets/datasets.py [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+It can also optionally download these datasets.
+
+To run this script:
+
+python3 datasets.py
+
+Be default, the script will only produce a CSV with the list of datasets.
+Once the CSV is created, to download the datasets, the download_datasets method will need to be called explicitly.
+
```
Comment (ajaits) [inline scripts/world_bank/datasets/datasets.py:24]: Can you also mention where files would be downloaded and what the output folder will contain?

File: scripts/world_bank/datasets/datasets.py (hunk 3 above)
Comment (keyurva) [reply scripts/world_bank/datasets/datasets.py]: Done.

File: scripts/world_bank/datasets/datasets.py (hunk 4 above)
Comment (keyurva) [reply scripts/world_bank/datasets/datasets.py:25]: Done.

**Review by ajaits — APPROVED** [review]

**Pull request merged**